### PR TITLE
PEP8 and formatting in optics.py

### PR DESCRIPTION
**Pull request by josePhoenix**
Some code polish and elbow grease for `optics.py`. Notably, this includes some class renames that were updated in `webbpsf_core.py` as well. I also moved the coordinate pairs for the JWST primary into a `constants.py` to improve the readability of `optics.py`. Here are the renames:

* `JWST_primary_aperture` -> `WebbPrimaryAperture`
* `JWST_OTE_Pupil` -> `WebbOTEPupil`
* `JWST_Field_Dependent_Aberration` -> `WebbFieldDependentAberration`
* `NIRSpec_Field_Dependent_Aberration` -> `NIRSpecFieldDependentAberration`
* `NIRCam_Field_and_Wavelength_Dependent_Aberration` -> `NIRCamFieldAndWavelengthDependentAberration`
* `MIRI_Field_Dependent_Aberration_and_Obscuration ` -> `MIRIFieldDependentAberrationAndObscuration`

(My small act of protest of always referring to JWST rather than Webb ran up against the aesthetics of naming something `JWSTOTEPupil` and my activist side lost.)

### CODE REVIEW

**Comment by mperrin** [comment]
> (My small act of protest of always referring to JWST rather than Webb ran up against the aesthetics of naming something JWSTOTEPupil and my activist side lost.)

Yeah, neither of those are great. I agree that "JWST" would be preferable in general (Overall it seems like people refer to the observatory less often as "Webb" these days than was common 5 or 6 years ago? But that's anecdotal, not data.)  But that stack of capitalized letters isn't aesthetically pleasing. Even so I think it might be better to just go with `JWSTOTEPupil` and the like.

**Comment by josePhoenix** [comment]
It is *WebbPSF*, after all. `WebbOTEPupil` is growing on me.

File: webbpsf/constants.py [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    'JWST_PRIMARY_SEGMENT_CENTERS',
+    'JWST_SEGMENT_RADIUS',
+    'JWST_CIRCUMSCRIBED_DIAMETER',
+)
+
+#
+# JWST Primary segment and obscuration shapes and centers
+#
+# Provenance: jwst_ote3d.py script by Perrin for personal use
+#
```
Comment (mperrin) [inline webbpsf/constants.py]: Provenance is rather:
```
Segment coordinates from "2010.03.16 Transmission X Area Budget.xls" spreadsheet by Paul 
Lightsey, which was based in turn on Ball Aerospace drawing # 2220169 Rev B and
the OTE Cryogenic Optics Interface Control Document, Ball Aerospace doc # C327693
```

Comment (josePhoenix) [inline webbpsf/constants.py]: Great, that's much better! Will update.

**Comment by mperrin** [comment]
> It is WebbPSF, after all. WebbOTEPupil is growing on me.

That reminds me of the question we've been dodging for a while, whether it should be instead `WebbAndWFIRSTPSF` (answer: no.) or something else entirely more inclusive (answer: maybe??) but rebranding would be a pain, so I'm still inclined to punt.

**Comment by josePhoenix** [comment]
Yes, let's kick that can down the road. (Maybe there's a Greek god or goddess of PSFs? Worked for the ETC people...)

**Pull request merged**